lib/AppInfo/Application.php

```diff
@@ -76,10 +76,7 @@ public function registerEventsScripts(IEventDispatcher $dispatcher) {
 		// 	\OCP\Util::addScript(self::APP_ID, 'dist/collaboration');
 		// });
 		//\OCP\Util::addScript(self::APP_ID, 'beeswarmfilelist' );
-<<<<<<< HEAD
-=======
-		\OCP\Util::addScript(self::APP_ID, "main");
->>>>>>> f2e5dee91ff6d550b63b77fbb990b8720d52fe99
+		//\OCP\Util::addScript(self::APP_ID, "main");
```

Comment:
New question, why is everything in this function commented

Reply:
Yes, it was due to tests. At this moment, there is no need to inject js scripts